.github/workflows/deploy.yml

```diff
@@ -0,0 +1,42 @@
+name: "Deploy site"
+
+on:
+  pull_request:
+    branches:
+    - '*'
+  push:
+    branches:
+    - 'master'
+
+jobs:
+  example-1:
+    name: Deploy lektor website
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10'
+      - name: Setup Python
+        uses: actions/setup-python@v1
```

Comment:
actions/setup-python@v2 is available

Reply:
Thanks! I hadn't changed anything yet and just rebased the old commit.